src/platform/packages/shared/kbn-esql-ast/src/__tests__/autocomplete.ts

```diff
@@ -75,9 +75,13 @@ export const suggest = (
 ): Promise<ISuggestionItem[]> => {
   const innerText = query.substring(0, offset ?? query.length);
   const correctedQuery = correctQuerySyntax(innerText);
-  const { ast } = parse(correctedQuery, { withFormatting: true });
+  const { ast, root } = parse(correctedQuery, { withFormatting: true });
+  const isHeaderInstruction = root?.header?.find((cmd) => cmd.name === commandName);
```

Comment:
```suggestion
  const headerInstruction = root?.header?.find((cmd) => cmd.name === commandName);
```

As this is the header command itself, not a boolean.